README.md

```diff
@@ -148,6 +148,13 @@ You can mark samples as drafts if they are still work-in-progress. This means th
 }--->
 ```
 
+#### How to style examples
+
+You can’t reference external stylesheets when creating AMP pages. All styles must live in the head of the document using the tag `<style amp-custom>`.
+For AMPByExample, we are using templates, so that we can manage better the CSS, as suggested [here](https://www.ampproject.org/docs/guides/responsive/style_pages.html).
+After you write the example and the page is rendered by gulp, the CSS classes will be added to the AMP page from the existing CSS files.
```

Comment:
I am trying to say what this file does https://github.com/ampproject/amp-by-example/blob/master/tasks/compile-example.js in the compileExamples function.